loader_hub/file/rdf/base.py

```diff
@@ -0,0 +1,65 @@
+"""Read RDF files."""
+
+from pathlib import Path
+from typing import Dict, List, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+
+from rdflib import Graph, URIRef
```

Comment:
Can you lazily import these from within the class? Makes testing easier and makes things cleaner when we download a bunch of loaders at once.

Reply:
I just saw you already did it. Thanks!